Dockerfile

```diff
@@ -1,5 +1,13 @@
 FROM centos:7
 
+
```

Comment:
I like this, but maybe add these comments directly onto the end of the install lines so we don't lose them in future rearrangements? So something like:
```
        readline-devel \ # needed for rpy2
        libcurl-devel \  # needed for rvest (for R)
```

Reply:
Done in: 

https://github.com/PrairieLearn/PrairieLearn/pull/1023/commits/12798b271b819f07809cc2b0c249b2d2d5a9caae#diff-03f6ff77580eb0ea364c4f941578c252R21